app/helpers/unicode_helper.rb

```diff
@@ -0,0 +1,6 @@
+module UnicodeHelper
+  def clear_invisible_string(str)
```

Comment:
__[Reek]__

[UtilityFunction] UnicodeHelper#clear_invisible_string doesn't depend on instance state (maybe move it to another class?)


<!-- Comment by [SideCI](https://sideci.com) -->